oap-server/server-starter/src/main/resources/application.yml

```
@@ -435,6 +435,17 @@ query-zipkin:
     # Default look back on the UI for search traces, 15 minutes in millis
     uiDefaultLookback: ${SW_QUERY_ZIPKIN_UI_DEFAULT_LOOKBACK:900000}
 
+query-prometheus:
```

```suggestion
promql:
```

From the official explanation, **PromQL, short for Prometheus Querying Language, is the main way to query metrics within Prometheus. You can display an expression's return either as a graph or export it using the HTTP API**. PromQL should be good enough.